Content.Shared/NameModifier/EntitySystems/NameModifierSystem.cs

```diff
@@ -32,6 +32,18 @@ private void SetBaseName(Entity<NameModifierComponent> entity, string name)
         Dirty(entity);
     }
 
+    /// <summary>
+    /// Returns the base name of the entity, without any modifiers applied.
+    /// If the entity doesn't have a <see cref="NameModifierComponent"/>,
+    /// this returns the entity's metadata name.
+    /// </summary>
+    public string GetBaseName(Entity<NameModifierComponent?> entity)
+    {
+        if (Resolve(entity, ref entity.Comp, logMissing: false))
+            return entity.Comp.BaseName;
+        return MetaData(entity).EntityName;
```

Comment:
```suggestion
        return Name(entity);
```
There is a shorthand for this!

Reply:
Thanks!